specs/0118-notification-center.md

```diff
@@ -0,0 +1,134 @@
+          - Datepicker: `To:` (Default value: today's date)
+      - Inputs for `Last login`
+        - `Timeframe filter`
+          - Datepicker: `From:` (Default value: today's date)
+          - Datepicker: `To:` (Default value: today's date)
+
+#### Flagging as Transactional
+
+- Header: `Transactional email`
+  - By default, emails are sent to users who have enabled opt-in consent. (Does not apply to in-app notifications.)
+  - This “Transactional” option flags emails as transactional and sends emails to users regardless of their opt-in status. However, this email content would be restricted to content transactional in nature (e.g. account status notifications, etc.).
+  - Checkbox (default: deselected): `This is a transactional email`
+  - Text description: `Transactional emails are sent to users who have not opted in to email communication. Transactional emails are restricted in the type of content that can be included.`
+  - Validation:
+    - Disable `Review message` button and show error message if `Transactional email` enabled but `Message type` does not include `Email`: `You cannnot enable the transactional email option if the Message type does not include email.`
+
+#### Proceed Button
+- Button: `Review message`
+  - A send button is not displayed on the same page as the message editor and recipient selection to avoid sending an incomplete message or a message to the wrong recipients.
+  - When pressed a validation step is performed on inputs to ensure there are no issues with the form. If there are no issues, the user proceeds to the Review/Confirmation page.
+
+### Review
+
+- Title: `Review message`
+- A review step is required to reduce errors because messages could be sent to a large number of people. This page allows you to review all aspects of the email before sending, including:
+ - A rendered version of the message to ensure things like Markdown tags or html don’t have issues.
+ - A summary of who will be receiving the nessage:
+    - The number of recipients
+    - The filters being applied
+  - Whether this is a transactional email. If enabled, a warning will be shown: `Warning: transactional emails are sent to users who have not opted in to email communication. Transactional emails are restricted in the type of content that can be included. When in doubt, please review with legal.`
+  - Button: `Send test to myself`
+    - Sends the current message to the user creating the message in the same format(s) as it is being sent.
+  - Button: `Edit`
+    - Clicking the ` Edit` button or navigating back in the browser brings you to the previous page
+    - The previously entered content is preserved when `Edit` or the browser's back button is used so work is not lost
+  - Button: `Send`
+    - Triggers validation step
+    - Initiates the sending of the email and/or notification
+
+## Email Content
+
+When sending an email to users via the Notification Center, emails include the following content:
```

Comment:
Nit: We should also rename "Notification Center" to "Messaging Center" in this section. Or even drop it?